app/medInria/medFilteringWorkspace.cpp

```diff
@@ -44,6 +44,7 @@ medFilteringWorkspace::medFilteringWorkspace(QWidget *parent): medAbstractWorksp
 {
     d->filteringToolBox = new medFilteringSelectorToolBox(parent);
     connect(d->filteringToolBox,SIGNAL(processFinished()),this,SLOT(onProcessSuccess()));
+    d->filteringToolBox->setWorkspace(this);
```

Comment:
In the case of medPipelineWorkspace I had to override the original addToolBox (because it only works when used in the constructor of a workspace and didn't fit our needs for the pipelines), and did the setWorkspace there.